src/jit/gentree.h

```diff
@@ -6022,6 +6048,40 @@ inline bool GenTree::IsMultiRegNode() const
 
     return false;
 }
+//-----------------------------------------------------------------------------------
+// GetMultiRegCount: Return the register count for a multi-reg node.
+//
+// Arguments:
+//     None
+//
+// Return Value:
+//     Returns the number of registers defined by this node.
+inline unsigned GenTree::GetMultiRegCount()
+{
+    if (IsMultiRegCall())
+    {
+        return AsCall()->GetReturnTypeDesc()->GetReturnRegCount();
+    }
+
+#if FEATURE_ARG_SPLIT
```

Comment:
I believe that this can safely be #if __TARGET_ARM__ instead of FEATURE_SPLIT_ARG. As arm64 will never have split return types even under windows.

Reply:
arm64 won't, but x86 can.

Reply:
But this isn't about return types, this is about arguments.

Reply:
Ah I misread this you can safely ignore my comment :)